app/javascript/packs/validateFileInputs.js

```diff
@@ -224,6 +287,13 @@ function validateFileInput(fileInput) {
       isValidFileInput = false;
       break;
     }
+
+    const isValidFileNameLength = validateFileNameLength(file, fileInput);
+
+    if (!isValidFileNameLength) {
```

Comment:
I see we have a pattern of storing this value in a constant above and then evaluating it in the if statement. Is there a reason we do it this way as opposed to `if (validateFileNameLength(file, fileInput))`?

Reply:
Nope - not a good reason at least! That's just a result of many refactors and personal preference. I'll change that 💯

Reply:
Oh don't worry about changing it! I just didn't know if there was a programmatic reason for it in javascript.